src/Workspaces/Core/Portable/CodeFixes/FixAllOccurrences/BatchSimplificationFixAllProvider.cs

```diff
@@ -40,6 +41,13 @@ protected virtual SyntaxNode GetNodeToSimplify(SyntaxNode root, SemanticModel mo
             return root.FindNode(diagnostic.Location.SourceSpan, findInsideTrivia: true);
         }
 
+        protected virtual Task<Document> AddSimplifyAnnotationsAsync(Document document, SyntaxNode nodeToSimplify, CancellationToken cancellationToken)
+        {
+            throw new NotImplementedException();
+        }
+
+        protected virtual bool NeedsParentFixup { get { return false; } }
```

Comment:
It returns false for FixAll provider for SimplifyTypeNames, which is a sub-type of BatchSimplificationFixAllProvider. For simplify type names, we just need to add simplify annotations to original node without needing parent fixup (original code path before this fix).